tests/foreman/api/test_syncplan.py

```diff
@@ -167,9 +175,52 @@ def test_create_sync_plan_with_invalid_random_name(self, name):
         @Feature: SyncPlan
 
         """
+        with self.assertRaises(HTTPError):
+            entities.SyncPlan(
+                name=name,
+                organization=self.org,
+            ).create()
 
+    @data(
+        '',
+        ' ',
+        None,
+    )
+    def test_create_sync_plan_with_no_enabled_value_negative(self, enabled):
+        """@Test: Create a sync plan with empty 'enabled' field.
 
-@stubbed()
+        @Assert: A sync plan can not be created with empty 'enabled' field.
+
+        @Feature: SyncPlan
+
+        """
+        with self.assertRaises(HTTPError):
+            entities.SyncPlan(
+                enabled=enabled,
```

Comment:
> What's the advantage of omitting a field in that way?

It depends on what you want to test. Do you want to test *invalid* values or *missing* values?

> We will have to manually specify some random interval, name, generate some date. […]

See second example, above.